sllurp/llrp.py

```diff
@@ -1198,11 +1345,23 @@ def resume(self, force_regen_rospec=False):
         d.addErrback(self.panic, 'resume() failed')
         self.send_ENABLE_ROSPEC(None, self.rospec['ROSpec'], onCompletion=d)
 
-    def sendLLRPMessage(self, llrp_msg):
-        assert isinstance(llrp_msg, LLRPMessage)
+    def sendMessage(self, msg_dict):
+        """Serialize and send a dict LLRP Message
+
+        Note: IDs should be modified in original msg_dict as it is a reference.
+        That should be ok.
```

Comment:
That wasn't from me. It's a commit from @fviard

Reply:
Yes, out of context the note could have been a little more explict I think.
This was to say that we will modify a element named 'ID' of the input msg_dict.
As the msg_dict provided in input is a dict, and so a reference, the input will be modified by this call to change the value of the 'ID' entry.
I don't think that anyone care, but still here is the warning.
So, when some other code will call this function, it can expect to have one element of the dict modified by the called function.